devices/profalux.js

```diff
@@ -20,6 +20,7 @@ module.exports = [
         zigbeeModel: [
             'MOT-C1Z06C\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000',
             'MOT-C1Z10F\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000',
+            'MOT-C1Z06F\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000',
```

Comment:
For what it's worth, I believe 06/10 are torque values. The motors exist in 06, 10, 20 and 30 Nm variants according to some specifications I've seen.
I /think/ the C/F are hardware revisions, but I am not certain.
I thought I would share that somewhere :)